.github/workflows/refresh-lockfiles.yml

```diff
@@ -0,0 +1,73 @@
+# This workflow periodically creates new environment lock files based on the newest
+# available dependency packages on conda-forge.
```

Comment:
Whether or not `conda-forge` is the source depends on the files within `requirements/ci`. Could we come up with a more agnostic statement here?

Reply:
Good point, will change